src/generics/utils.js

```diff
@@ -552,6 +591,122 @@ function validateFilters(input, validationData, modelName) {
 	return input
 }
 
+const algorithm = process.env.EMAIL_ID_ENCRYPTION_ALGORITHM
+const secretKey = Buffer.from(process.env.EMAIL_ID_ENCRYPTION_KEY, 'hex')
+const fixedIV = Buffer.from(process.env.EMAIL_ID_ENCRYPTION_IV, 'hex')
+
+const encrypt = (plainTextEmail) => {
```

Comment:
These cannot be in mentoring service. The encryption and decryption should be handled by user service.

Reply:
@nevil-mathew Will change it .

Reply:
User service always accepts plain text email right?
You can ref /login

Reply:
User service always accepts plain text email, right?
You can ref /login

Reply:
@nevil-mathew I have chnaged it and , i am sending plainTextEmail to user service and encrypting in user services.